src/unix/notbsd/linux/mod.rs

```diff
@@ -407,7 +409,9 @@ extern {
     pub fn tmpfile64() -> *mut ::FILE;
     pub fn fgetpos64(stream: *mut ::FILE, ptr: *mut fpos64_t) -> ::c_int;
     pub fn fsetpos64(stream: *mut ::FILE, ptr: *const fpos64_t) -> ::c_int;
+    #[cfg(not(any(target_arch = "powerpc64", target_arch = "powerpc64le")))]
     pub fn fstat64(fildes: ::c_int, buf: *mut stat64) -> ::c_int;
```

Comment:
They aren't implemented in the Linux PowerPC64 kernel, but there are glibc wrappers that fall back to stat() etc. I'll add  stat64 etc back in for completeness.